code/modules/mob/living/simple_animal/abnormality/he/pinocchio.dm

```diff
@@ -0,0 +1,274 @@
+		ABNORMALITY_WORK_INSIGHT = 0,
+		"Atachment" = 0,
+		"Represion" = 0,
+		"Lying is Bad!" = 100
+	),
+		"lie3" = list(
+		"Insignt" = 0,
+		ABNORMALITY_WORK_INSIGHT = 0,
+		"Attachnent" = 0,
+		ABNORMALITY_WORK_REPRESSION = 0,
+		"Lying is Bad!" = 100
+	),
+		"normal" = list(
+		ABNORMALITY_WORK_INSTINCT = 45,
+		ABNORMALITY_WORK_INSIGHT = 45,
+		ABNORMALITY_WORK_ATTACHMENT = 45,
+		ABNORMALITY_WORK_REPRESSION = 45,
+		"Lying is Bad!" = 0
+	)
+	)
+
+//Spawn
+/mob/living/simple_animal/hostile/abnormality/pinocchio/PostSpawn()
+	..()
+	alpha = 0
+	pixel_z = 16
+	animate(src, alpha = 255,pixel_x = 0, pixel_z = -16, time = 4 SECONDS)
+	pixel_z = 0
+
+//Work/Misc
+/mob/living/simple_animal/hostile/abnormality/pinocchio/AttemptWork(mob/living/carbon/human/user, work_type)
+	if(realboy)
+		to_chat(user, "<span class='warning'>The abnormality isn't in here!</span>")
+		return FALSE
+	if(work_type == "Lying is Bad!")
+		if(lying)
+			playsound(src, 'sound/abnormalities/pinocchio/success.ogg', 40, 0, 1)
+			caught_lie = TRUE
+			lying = FALSE
+			datum_reference.qliphoth_change(1)
+			PostWorkEffect()
+			return
+		else
+			datum_reference.qliphoth_change(-1)
+		return FALSE
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/pinocchio/PostWorkEffect(mob/living/carbon/human/user, work_type, pe)
+	if(lying)
+		playsound(src, 'sound/abnormalities/pinocchio/lie.ogg', 40, 0, 1)
+		datum_reference.qliphoth_change(-1)
+		new /obj/effect/temp_visual/pinocchio/lies(get_turf(src))
+		return
+	if(caught_lie)
+		caught_lie = FALSE
+		var/list/new_work_chances = modular_work_chance["normal"]
+		work_chances = new_work_chances.Copy()
+		datum_reference.available_work = work_chances
+		new /obj/effect/temp_visual/pinocchio/caught(get_turf(src))
+		return
+	playsound(src, 'sound/abnormalities/pinocchio/activate.ogg', 40, 0, 1)
+	new /obj/effect/temp_visual/pinocchio(get_turf(src))
+	if(!lying && prob(30))
+		var/lienumber = pick("lie1","lie2","lie3")
+		var/list/new_work_chances = modular_work_chance[lienumber]
+		lying = TRUE
+		work_chances = new_work_chances.Copy()
+		datum_reference.available_work = work_chances
+
+//Breach
+/mob/living/simple_animal/hostile/abnormality/pinocchio/ZeroQliphoth(mob/living/carbon/human/user)
+	playsound(src, 'sound/abnormalities/pinocchio/activate.ogg', 40, 0, 1)
+	animate(src, alpha = 0,pixel_x = 0, pixel_z = 16, time = 4 SECONDS)
+	SLEEP_CHECK_DEATH(1 SECONDS)
+
+	realboy = new (get_turf(src)) //Technically the breach version is a separate entity, requires a lot of tinkering but works.
+	RegisterSignal(realboy, COMSIG_LIVING_DEATH, .proc/PuppetDeath)
+	realboy.name = "Pinocchio the Liar"
+	realboy.real_name = "Pinocchio the Liar"
+	realboy.ai_controller = /datum/ai_controller/insane/murder/puppet
+	realboy.InitializeAIController()
+	realboy.apply_status_effect(/datum/status_effect/panicked_type/puppet)
+	realboy.adjust_all_attribute_levels(100)
+	realboy.adjust_attribute_buff(FORTITUDE_ATTRIBUTE, 520) // 600 health
```

Comment:
720 health. Still fairly weak imo.

Reply:
He now spawns with a HE level E.G.O. We may need to consider toning it further down, now.